src/components/Summary/AlterationSummary.vue

```diff
@@ -272,6 +277,21 @@ export default class AlterationSummary extends Mixins(
     await this.scrollToTop(document.getElementById('app'))
   }
 
+  // sum of alteration fees
+  get alterationFees (): string {
+    if (this.getCurrentFees?.filingFees) {
+      return `($${(this.getCurrentFees.filingFees + this.getCurrentFees?.futureEffectiveFees).toFixed(2)} Fee)`
+    }
+    return ''
+  }
+
+  get futureEffectiveFeePrice (): string {
+    if (this.getFeePrices?.futureEffectiveFees) {
+      return `of $${this.getFeePrices.futureEffectiveFees.toFixed(2)}`
```

Comment:
Does getFeePrices always return an object? Or sometimes null?

Reply:
Yes, it always returns an object but I prefer being safe here, as someone can change it in the future.

Reply:
OK, yes, I see the "if" now :)

I guess you don't need the "?" on line 283 for the same reason then.

Reply:
ok.

Reply:
What happens if futureEffectiveFees is 0 here?

Reply:
ops... update here as well. Thanks

Reply:
fixed